### PR TITLE
KBI 0029: Create an empty RIA store

### LINKED ISSUE #91: RIA questions: https write access? create empty store without a dataset?
Origin: DataLad matrix channel, thread started 2023-06-19

Discussion is ongoing so won't copy-paste now, but we are discussing the following questions:
1. Is it possible to have an RIA with https read/write access?
2. Is it possible to create an empty RIA directly without first making a regular datalad repo?
3. Is it also possible to create an empty dataset within the RIA

TODO (not necessarily to be performed in this order)
- [ ] Inform OP/Add reference to this issue at origin
- [x] Clarifying Qs asked or not needed
- [x] Nature of the issue is understood
- [ ] Inform OP about resolution

### PR DESCRIPTION
**Pull request by loj**
A write-up demonstrating how to create an empty RIA store by hand or with the helper function `create_store` in `datalad/customremotes/ria_utils.py`.

Closes #91

### CODE REVIEW

File: kbi/0029/index.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
+.. index::
+   single: datalad; RIA
+.. highlight:: console
+
+KBI0029: Create an empty RIA store
+==================================
+
+:authors: Laura Waite <laura@waite.eu>
+:discussion: <link>
```
Comment (loj) [inline kbi/0029/index.rst:9]:
```suggestion
:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/110
```

File: kbi/0029/index.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  update(ok): . (dataset)
+  update(ok): . (dataset)
+  [INFO   ] Configure additional publication dependency on "ria-storage"
+  configure-sibling(ok): . (sibling)
+  create-sibling-ria(ok): /tmp/my_dataset (dataset)
+  action summary:
+    configure-sibling (ok: 1)
+    create-sibling-ria (ok: 1)
+    update (ok: 1)
+  0.00 [00:00, ?/s]
```
Comment (adswa) [inline kbi/0029/index.rst:81]:
```suggestion

```

**Review by adswa — APPROVED** [review]
This works for me! Thanks! I just removed an irrelevant line of logging.

**Review by jsheunis — APPROVED** [review]
Everything works on my side 👍 

let's hit the button

**Pull request merged**